tests/rados/test_osd_db_wal_configuration.py

```diff
@@ -0,0 +1,207 @@
+import random
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    log.info(run.__doc__)
+    cephadm = CephAdmin(cluster=ceph_cluster)
+    rados_object = RadosOrchestrator(node=cephadm)
+    try:
+        available_device_list = rados_object.get_orch_device_list()
+        number_available_ssd = count_available_SSD(available_device_list)
+        if number_available_ssd == 0:
+            log.error("The SSD/NVME devices are not exist to perform the tests")
+            return 1
+        log.info(f"The total SSDs exist in the cluster are-{number_available_ssd}")
+        available_ssd_path = get_ssd_path(available_device_list)
+        log.info(f"The available SSD ins the cluster are-{available_ssd_path}")
+
+        ceph_nodes = kw.get("ceph_nodes")
+        osd_nodes = []
+        for node in ceph_nodes:
+            if node.role == "osd" and node.hostname in available_ssd_path:
+                osd_nodes.append(node)
+        test_count = 0
+
+        for osd_node in osd_nodes:
+            ssd_paths = available_ssd_path[osd_node.hostname]
+            for path in ssd_paths:
+                # Creating Volume group
+                cmd_vgcreate = f"vgcreate LVMTest {path}"
+                log.info(
+                    f"Creating volume group in {osd_node.hostname} osd node at {path}"
+                )
+                vg_out, _ = osd_node.exec_command(
+                    sudo=True, cmd=cmd_vgcreate, check_ec=False
+                )
+                log.info(f"The volume creation output is {vg_out}")
+                if "successfully created" not in vg_out:
+                    log.info(
+                        f"Volume group is not created in the {osd_node.hostname}.Skipping the further tests."
+                    )
+                    continue
+                log.info(
+                    f"Volume group is created at {osd_node.hostname} osd node at {path}"
+                )
+                # Create logical volume
+                log.info("Creating logical volumes")
+                cmd_lvcreate_base = "lvcreate -l 5 -n"
```

Comment:
What is the size of LV's being added here? 

I strongly recommend that we get the size of the OSD device, and only create LV's of recommended size, i.e 4% of data device.